# Description: don't list mozilla commands which are compatibility symlinks
# Ubuntu: http://bugs.launchpad.net/bugs/55963
# Upstream: http://bugzilla.gnome.org/show_bug.cgi?id=499327
diff -Nur gnome-control-center-2.19.1/capplets/default-applications/gnome-default-applications.xml.in gnome-control-center-2.19.1.new/capplets/default-applications/gnome-default-applications.xml.in
--- gnome-control-center-2.19.1/capplets/default-applications/gnome-default-applications.xml.in	2007-05-02 17:39:17.000000000 +0200
+++ gnome-control-center-2.19.1.new/capplets/default-applications/gnome-default-applications.xml.in	2007-05-02 17:40:38.000000000 +0200
@@ -82,26 +82,6 @@
       <win-command>iceweasel -remote "openurl(%s,new-window)"</win-command>
     </web-browser>
     <web-browser>
-      <_name>Mozilla 1.6</_name>
-      <executable>mozilla-1.6</executable>
-      <command>mozilla-1.6 %s</command>
-      <icon-name>mozilla-icon</icon-name>
-      <run-in-terminal>false</run-in-terminal>
-      <netscape-remote>true</netscape-remote>
-      <tab-command>mozilla-1.6 -remote "openurl(%s,new-tab)"</tab-command>
-      <win-command>mozilla-1.6 -remote "openurl(%s,new-window)"</win-command>
-    </web-browser>
-    <web-browser>
-      <_name>Mozilla</_name>
-      <executable>mozilla</executable>
-      <command>mozilla %s</command>
-      <icon-name>mozilla-icon</icon-name>
-      <run-in-terminal>false</run-in-terminal>
-      <netscape-remote>true</netscape-remote>
-      <tab-command>mozilla -remote "openurl(%s,new-tab)"</tab-command>
-      <win-command>mozilla -remote "openurl(%s,new-window)"</win-command>
-    </web-browser>
-    <web-browser>
       <_name>SeaMonkey</_name>
       <executable>seamonkey</executable>
       <command>seamonkey %s</command>
@@ -241,13 +206,6 @@
       <run-in-terminal>false</run-in-terminal>
     </mail-reader>
     <mail-reader>
-      <_name>Mozilla Mail</_name>
-      <executable>mozilla</executable>
-      <command>mozilla -mail %s</command>
-      <icon-name>mozilla-mail-icon</icon-name>
-      <run-in-terminal>false</run-in-terminal>
-    </mail-reader>
-    <mail-reader>
       <_name>SeaMonkey Mail</_name>
       <executable>seamonkey</executable>
       <command>seamonkey -mail %s</command>
